### ISSUE TITLE
AL 164674 (centre)

### ISSUE THREAD

**Issue by annabirnbaum** [precision-sustainable-ag/On-farm-Protocols#88]
Kip called me directly: Gateway 164674 is only sending gateway data since 6/5/21. He is going to try raising the gateway. Depth 80 of node 318 is ready -999.  We walked through some troubleshooting ideas and he will call if when he is ready to use the test function on nodes to test if sensor is corrected.

**Comment by annabirnbaum** [comment]
He removed depth 80 from ground,  sees no wire damage, walked him through T function, not bent.  He didn't have bucket of water to test in water, but we stabbed it in the ground surface and still -999, in air= still -999.  I think he needs a replacement sensor -80 bare plot rep 2 @elhenriq 

In the image attached, he noticed the ^M^@ after the C13 reading.  I've noticed this in another sensor reading -999.  Maybe this is a a clue? 

He tried raising the gateway a few feet and it still hasn't connected yet...

![IMG_0098](https://user-images.githubusercontent.com/64480167/121074255-99dd2080-c7a1-11eb-8bba-3c72eaabc5cd.JPG)
